### DISCUSSION TITLE
Fixed LTSC 2024

### DISCUSSION THREAD

**Discussion by TheMMC** [TheMMC/In-Place_Upgrade_Helper#15] [Announcements]
Option 21 and 23 were identical by mistake

<hr /><em>This discussion was created from the release <a href='https://github.com/TheMMC/In-Place_Upgrade_Helper/releases/tag/v0.93'>Fixed LTSC 2024</a>.</em>

**Comment by Trubidubidu** [comment]
![Programmausgabe](https://github.com/user-attachments/assets/96d8d92d-5e31-4902-b51e-7073e6465a05)
Hello, is it possible to make an in-place-upgrade from Windows 10 Pro 22H2 to Windows 10 Enterprise N LTSC 2019 ? I made a test (I have the correct sources and key of Windows 10 Enterprise N LTSC 2019). But when I start your program, the program only gives me the LTSC 2021 version (among other versions) to choose from. This version LTSC 2021 has a major disadvantage: it will only receive updates for five years (until 2026). The 2019 version will receive updates for ten years (until 2029).
But as I said: although I have the correct source and start the program from this source, it does not show me this version to choose from.
Do you have any idea how to solve the problem? Because the in-place upgrade to the LTSC 2019 version is particularly important.
Anyway, thank you very much for the great program.
Antwort gerne auch auf deutsch.

**Reply by TheMMC** [reply]
Where did you get your sources and what is the name of that ISO?
LTSC 2021 and LTSC 2019 use completely different sources and builds, a 2019 ISO simply cannot start a 2021 setup:
https://massgrave.dev/windows_ltsc_links

**Reply by Trubidubidu** [reply]
Thank you for your very quick and competent response. Thank you also for your link. 
I used this version, but apparently it was the wrong one:
https://vollversionkaufen.de/sl-win10-ent-n-ltsc-2019-64b
If I take the sources from the correct version, will the upgrade work with your program?
Or is that a stupid question on my part?

**Reply by TheMMC** [reply]
Regarding the updates:
Listing - Start Date - Mainstream End Date - Extended End Date
Windows 10 Enterprise LTSC 2019 - Nov 13, 2018 - Jan 9, 2024 - Jan 9, 2029
Windows 10 IoT Enterprise LTSC 2019 - Nov 13, 2018 - Jan 9, 2024 - Jan 9, 2029
Windows 10 Enterprise LTSC 2021 - Nov 16, 2021 - Jan 12, 2027 - -
Windows 10 IoT Enterprise LTSC 2021 - Nov 16, 2021 - Jan 12, 2027 - Jan 13, 2032
Windows 11 Enterprise LTSC 2024 - Oct 1, 2024 - Oct 9, 2029 - -
Windows 11 IoT Enterprise LTSC 2024 - Oct 1, 2024 - Oct 9, 2029 - Oct 10, 2034

Windows 22H2 to LTSC 2019 is technically a downgrade (build 19045 to 17763), are you sure you want to do this? Downgrades are untested and could go wrong. I would recommend Windows 10 IoT LTSC 2021 if you want to stay on 1904X. Or even Windows 11 IoT LTSC 2024, depending on your hardware. The Windows 11 LTSC is the new germanium kernel with the speed optimizations for the new rizen cpus and such, so choose wisely.

I just tried doing an downgrade from 22H2 to LTSC 2019, I couldnt get it to work, Windows Setup won't do downgrades.
But you could do an upgrade to LTSC 2021:
![grafik](https://github.com/user-attachments/assets/298d6e14-538b-4d71-8507-1a48e52e027d)
After that, do a simple key upgrade to IoT LTSC. Now you have updates until 2032.

**Reply by Trubidubidu** [reply]
Hello, that was very nice of you. Thank you for your excellent thoughts. You helped me perfectly. Your expertise is impressive. I hope you have a nice day.
